src/dotnet/Properties/AssemblyInfo.cs

```diff
@@ -3,3 +3,4 @@
 
 [assembly: AssemblyMetadataAttribute("Serviceable", "True")]
 [assembly: InternalsVisibleTo("dotnet.Tests")]
+[assembly: InternalsVisibleTo("Microsoft.DotNet.ProjectJsonMigration.Tests")]
```

Comment:
Yes it is the end to end tests. I like calling `dotnet migrate` better than what they're currently doing, that should let me remove the internals visible to.